### DISCUSSION TITLE
COLLADA. Import problem.

### DISCUSSION THREAD

**Discussion by Ave23** [assimp/assimp#4004] [General]
![image](https://user-images.githubusercontent.com/40492748/127321543-ae24e1b8-1164-406f-b9b6-4b6b2bacb80c.png)
The import file is completed by an error if the author is written as follows:
  < Author  />"

**Comment by kimkulling** [comment]
Thanks for the hint. I have generated an issue report out of this.

**Comment by kimkulling** [comment]
The problem is fixed on the current master. The handling for empty XML-nodes with irrXML was broken. With pugixml it works.

**Reply by Ave23** [reply]
> The problem is fixed on the current master. The handling for empty XML-nodes with irrXML was broken. With pugixml it works.

Thank you for your quick response. Apparently, the release of a new stable version of the library will not be soon?
